### PR TITLE
DOC-544 - Create stores section

### PR DESCRIPTION
**Pull request by diana-macrometa**
- Move Stores functions to Stores folder
- Created Create Stores index.md topic. @StoyanTc, please give this a thorough review
- Deleted Procedure section from RDBMS topic

The Table topic will be moved to this new section in a later PR.

### CODE REVIEW

File: docs/cep/store/index.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+CREATE TABLE SensorTable (sensorId string, temperature double);
+```
+
+## Supported Stores
+
+Supported stores include:
+
+- [Macrometa collections](../../collections/index.md) (also called tables)
+- [MongoDB](mongodb.md)
+- [Redis](redis.md)
```
Comment (StoyanTc) [inline docs/cep/store/index.md]: @diana-macrometa - We don't support `MongoDb` and `Redis` yet.

File: docs/cep/store/index.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+- [RDBMS](rdbms.md)
```
Comment (StoyanTc) [inline docs/cep/store/index.md]: @diana-macrometa - RDBMS is too common. We do support `MySQL`, `PostgresQL`, `Oracle`.

**Review by StoyanTc — COMMENTED** [review]
Added few details in `index.md`.

File: docs/cep/store/index.md (hunk 2 above)
Comment (diana-macrometa) [reply docs/cep/store/index.md]: What do you mean by "too common"? Are you saying that MySQL, PostgresQL, and Oracle are types of RDBMS? Do we need to make separate documentation for each of these?

Comment (StoyanTc) [reply docs/cep/store/index.md]: @diana-macrometa - There are many RDBMS databases. We support only those from above and `SQLServer`.

Comment (diana-macrometa) [reply docs/cep/store/index.md]: Updated. Please take a look.

**Review by StoyanTc — APPROVED** [review]
Looks good now.

**Pull request merged**